src/Button/__tests__/BorderlessButton.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'test-utils';
+import { spy } from 'sinon';
+import assert from 'assert';
+import BorderlessButton from '../BorderlessButton';
+
+describe('Borderless Button', function () {
+  let getByText;
+  let clickSpy;
```

Comment:
These should be typed (e.g. `let clickSpy: SinonSpy;`)

_This applies to other button tests as well._

Reply:
Fixed through d6fad8a